tools/integration_tests/log_content/file_upload_log_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log_content
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"path"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
+)
+
+const (
+	BigFileSize              int64 = 50 * operations.MiB
+	SmallFileSize            int64 = operations.MiB
+	DirForBigFileUploadLog         = "dirForiBigFileUploadLog"
+	DirForSmallFileUploadLog       = "dirForSmallFileUploadLog"
```

Comment:
this constant is not used. you are using BigFileUpload directory for  small file also. in that case rename DirForBigFileUploadLog to DirForFileUploadLog

Reply:
This was a copy-paste error when I tried to extract out the common logic for logString from the two tests. 

It would be neater as you suggested, but it will also need changes if we ever tried to parallelize the two tests with each other, as both will try writing to same file/folder and will crash. So, I've separated them in two separate folders.